packages/color-scale-block/src/dragAndDrop/DropZone.tsx

```diff
@@ -0,0 +1,74 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useDrop } from 'react-dnd';
+import { DraggableItem, DropZonePosition, OrderableListItem, merge } from '@frontify/fondue';
+import { DropZoneProps } from '../types';
+
+export const DropZone = <T extends object>({
+    data,
+    currentColor,
+    isDraggingActive,
+    onDrop,
+    children,
+    treeId,
+    height,
+}: DropZoneProps<T>) => {
+    const [{ isOver, canDrop }, drop] = useDrop({
+        accept: treeId,
+        drop: (item: OrderableListItem<T>) => {
+            onDrop?.(data.targetItem as DraggableItem<T>, item, data.position);
+        },
+        canDrop: (item: OrderableListItem<T>) => {
+            // can't drop an item on itself
+            return !(item.id === data.targetItem.id);
+        },
+        collect: (monitor) => ({
+            isOver: monitor.isOver(),
+            canDrop: monitor.canDrop(),
+        }),
+    });
+
+    const width = currentColor && currentColor.width;
+
+    const isActive = isOver && canDrop;
+
+    // When dragging is active but this square is not being hovered over
+    const isDraggingActiveClassNames = 'tw-absolute tw-w-full tw-z-[100]';
+
+    // When no dragging is happening
+    const outerDropZoneClassNames = 'tw-absolute tw-w-full tw-py-1 tw-outline-none';
+    const isDraggingNotActiveClassNames = 'tw-z-[-1] tw-border-0';
+
+    // When dragging is happening and another square is hovered over this one
+    const activeOuterDropZoneClassNames = 'tw-absolute tw-z-[100] tw-h-[96px] tw-w-full tw-bg-clip-content';
+    const bgColorClassName = 'tw-bg-violet-20';
+
+    return (
+        <>
+            <div
+                style={{
+                    width: isActive ? `${parseInt(width)}px` : '0px',
+                    height: `${height - 2}px`,
+                }}
+                className={`drop-zone-offset tw-relative tw-transition-all tw-rounded-[3px] tw-bg-[#E3E8F6] ${
```

Comment:
Use `merge` from `Fondue for conditional classes. 

Also `tw-bg-[#E3E8F6]` is not allowed, either find the color you need in Fondue or define it in the `tailwind.config`.